services/line-stat-getter.js

```diff
@@ -172,7 +200,7 @@ function LineStatGetter(model, params, opts) {
 
   this.perform = function () {
     return model.unscoped().findAll({
-      attributes: [getGroupByDateInterval(), getAggregate()],
+      attributes: [getGroupByDateInterval(), getAggregate(), getGroupByDateFormat()],
```

Comment:
Really, there is not reason to pass an hardcoded date format in the SQL query.
Anyway it does not work on a Postgres query:
`SequelizeDatabaseError: column "YYYY-MM-DD" does not exist`

And if you run the tests, some of them fail for that reason.

Reply:
Shit, sorry about the regression. 

If you think we don't need to hardcode de format in the SQL query, can you suggest me a solution?